cli/pkg/local/app.go

```diff
@@ -272,6 +273,9 @@ func (a *App) Serve(httpPort, grpcPort int, enableUI, openBrowser, readonly bool
 			mux.Handle("/local/config", a.infoHandler(inf))
 			mux.Handle("/local/version", a.versionHandler())
 			mux.Handle("/local/track", a.trackingHandler(inf))
+			mux.Handle("/v1/downloads", &downloads.DownloadHandler{
+				Runtime: a.Runtime,
+			})
```

Comment:
This download API won't just be for local, but also in cloud. So mount it around `runtime/server/server.go:165` instead